server/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoModel = require('../database/index');
+const app = express();
+
+
+
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+//REMEMBER TO ADD STATIC PATH
+
+//REMEMBER TO FIX FIND ONE BELOW!!!!!!!!!!!!
+app.get('/videoData', (req, res) => {
```

Comment:
Refactor to use url params '/videoData/:id'

Reply:
Consider renaming endpoint '/mediaData' so that the name encompasses both videos and images